index.html

```diff
@@ -472,7 +472,9 @@ <h4>Workspace</h4>
                                 <option value="5">Auto</option>
                             </select>
                         </div>
-
+                        <div id="spectrumExport" data-toggle="tooltip" title="Export spectrum to CSV">
+                            <button id="btn-spectrum-export" type="button">Export to Csv</button>
```

Comment:
This is the last one :see_no_evil: 
```suggestion
                            <button id="btn-spectrum-export" type="button">Export to CSV</button>
```